code-challenges.js

```diff
@@ -22,12 +132,76 @@ var middleLetters1 = "hello"
 // Expected output: “l”
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
-
-
-
+var middleLetters3 = "yeot"
+var middleLetters4 = "herliktomt"
+var middleLetters5 ="thisisanevennumber"
+//the funtion takes in stirngs that are words
+//if word is even return two middle middleLetters
+//else if return one middle letters
+//.length in order to count the the middle
+//use charAt() to get the middle letter
+
+//function takes in strings
+const middleFinger = (string) => {
+  //need to find if string length is odd or even
+  let lengthFinder = string.length
+  if(lengthFinder % 2 != 0){
+    //return charAt for the middle
+    return string.charAt(lengthFinder *.5)
+  }else{
+    //return the middle letter and the one before that for two middle ones
+    return string.substring(lengthFinder * .5 - 1, lengthFinder * .5) + string.charAt(string.length*.5)
+    //substring
+  }
+}
+
+
+
+
+
+
+// const middleFinder = (string) => {
+//   let stringer = string.length
+//   if(string.length % 2 === 0 ){
+//     return charAt(string.length)
+//   } else if(string.length % 2 !== 0){
+//     return "yeet"//charAt(string.length * .5 && string.length *.5 + 1)
+//   }
+// }
+//
+ console.log(middleFinger(middleLetters1));
+ console.log(middleFinger(middleLetters2));
+ console.log(middleFinger(middleLetters3));
+ console.log(middleFinger(middleLetters4));
+ console.log(middleFinger(middleLetters5));
+//
 
 // --------------------4) READ CAREFULLY: Create a CLASS to get the area of a sphere. Create THREE spheres with different radi as test cases. Area of a sphere =  4πr^2 (four * pi * radi squared)
 
+//the four and pie reamin constant
+//need to create a variable for radisu and must make radius multiply with itself
+//need to understand what classes are and how they work
+
+//need a class
+//pi and four are constant
+class Area{
+  constructor(constant, radius){
+    this.constant = 12.56
+    this.radius = radius
+  }
+  //method will carry out all the math.
+  area (){
+    return `${this.constant * this.radius * this.radius}`
+ }
+}
+var sphere = new Area(12.56, 5)
+var sphere2 = new Area(12.56, 10)
+var sphere3 = new Area(12.56, 21)
+var sphere4 = new Area(12.56, 15)
+console.log(sphere.area());
+console.log(sphere2.area());
+console.log(sphere3.area());
+console.log(sphere4.area());
 
```

Comment:
Interesting approach with the 4*Pi. It gets the job done. One note, since you already have constant declared in the class and it is, by your own definition, constant :) I think you can skip the part where you pass in 12.56 in the creation of your new objects.